ros_buildfarm/status_page.py

```diff
@@ -1030,3 +1034,39 @@ def _compare_package_version(distros, pkg_name):
         data[i] = '<div>%s</div>' % value
 
     return data
+
+
+REPOS_DATA_NAMES = ['build', 'test', 'main']
+def write_summary(summary_filename, ordered_pkgs, repos_data):
```

Comment:
Maybe rename this to `write_yaml` to clarify what the function is doing. Otherwise the reader might expect it printing a line with some statistics.